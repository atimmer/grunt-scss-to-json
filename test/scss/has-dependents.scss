$first: $blue;
$global-variable: $red !global;
$references: $first;
$scss-function-with-variable: quote($red);
