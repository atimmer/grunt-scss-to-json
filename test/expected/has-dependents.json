{"$first":"#00f","$global-variable":"#f00","$references":"#00f","$scss-function-with-variable":"\"#f00\""}
